tests/scoring/test_scorer.py

```diff
@@ -25,10 +29,26 @@ def test_score_collections_string_attributes(self):
             ]
         )
 
-        scorer = OpenRarityScorer()
-        scores = scorer.score_collection(collection)
+        collection_two = generate_collection_with_token_traits(
+            [
+                {"bottom": "1", "hat": "1", "special": "true"},
+                {"bottom": "1", "hat": "1", "special": "false"},
+                {"bottom": "2", "hat": "2", "special": "false"},
+                {"bottom": "2", "hat": "2", "special": "false"},
+                {"bottom": "pants", "hat": "cap", "special": "false"},
+            ]
+        )
+
+        scores = self.scorer.score_collection(collection)
         assert len(scores) == 5
 
+        scores = self.scorer.score_tokens(collection, collection.tokens)
+        assert len(scores) == 5
+
+        scores = self.scorer.score_collections([collection, collection_two])
+        assert len(scores[0]) == 5
+        assert len(scores[1]) == 5
```

Comment:
Should there be a test for expected score value as well? With so few tokens it should be easy to validate.

Reply:
We do have test with fixed scores - check the ranker PR that i posted